Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader

```diff
@@ -0,0 +1,86 @@
+﻿/************************************************************************************
+
+Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.  
+
+See SampleFramework license.txt for license terms.  Unless required by applicable law 
+or agreed to in writing, the sample code is provided “AS IS” WITHOUT WARRANTIES OR 
+CONDITIONS OF ANY KIND, either express or implied.  See the license for specific 
+language governing permissions and limitations under the license.
+
+************************************************************************************/
+
+Shader "Custom/Procedural Gradient Skybox"
```

Comment:
Please change the Shader folder to be "MRTK-Quest/Procedural Gradient SkyBox" to differentiate it from the source shader.

Reply:
Done